### ISSUE TITLE
custom width nd height not working

### ISSUE THREAD

**Issue by shanvpz** [zetbaitsu/Compressor#121]
when i tried custom compress, the height and widths are not changing, file size ok.

**Comment by lewkka** [comment]
Yeah, looks like they just ignored. For my case width and height changed, but not in the numbers I set for Compressor.

**Comment by zihadrizkyef** [comment]
same

**Comment by mtrezza** [comment]
Same

**Comment by mtrezza** [comment]
The parameters for `setMaxWidth` and `setMaxHeight` work differently than their names may suggests.

They define the **minimum required width and height** of the compressed image. Downsampling happens by the power of 2, so the image size will be divided by 2 until the minimum height or weight is reached.

**Comment by Shouheng88** [comment]
you can try my library: https://github.com/Shouheng88/Compressor

**Comment by 13242445359** [comment]
这个问题还没解决吗？
